### ISSUE TITLE
ComfyUI-Impact-Pack is always slow to start

### ISSUE THREAD

**Issue by julien-blanchon** [ltdrdata/ComfyUI-Impact-Pack#534]
Hi, `ComfyUI-Impact-Pack` is by far the slowest custom_nodes I've in my setup, it can take between 2sec and 3sec to launch.
It's not a lot for many users but for my specific need it's still important, I need to be able to spin instance of comfy very fast in a serverless fashion.

Did you know what slow down `ComfyUI-Impact-Pack` at startup and how I can fix that ?

The only Impact-Pack output I have are:
```raw
### Loading: ComfyUI-Impact-Pack (V4.85.1)
### Loading: ComfyUI-Impact-Pack (Subpack: V0.4)
[Impact Pack] Wildcards loading done.
```

**Comment by ltdrdata** [comment]
Does what you refer to as 2-3 seconds mean that it is indicated like this?
`   0.8 seconds: /mnt/teratera/git/ComfyUI/custom_nodes/ComfyUI-Impact-Pack`

**Comment by julien-blanchon** [comment]
Yes, I don't know why it's that slow. I initially tought that was related to the `impact-pack.ini` file. It seems to be the case on my local computer, so I've add the `impact-pack.ini` directly in the docker image but it's still slow to boot on the cloud instance.

**Comment by ltdrdata** [comment]
1. Have you changed settings like `mmdet_skip` to false? The `mmdet` module is very slow to import.
2. Are you using large wildcard files? Using large wildcard files can take a long time.

**Comment by julien-blanchon** [comment]
`mmdet_skip` is set to true.
I'm using the default wildcard file
